### DISCUSSION TITLE
Fails to load pre-existed world

### DISCUSSION THREAD

**Discussion by NerdyTrout** [Pryaxis/TShock#2590] [Q&A]
I tried to load my world, but each time it fails to load and Tshock just closes in instant. I can easily start my world in singplayer or multiplayer. Every new world i create via Tshock console seems to load just fine.
I got this error in crashlog:

3/7/2022 2:12:21 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at Terraria.Main.WriteFancyWorldLoadErrorToConsole()
   at Terraria.Main.DedServ()
   at Terraria.Program.LaunchGame(String[] args, Boolean monoArgs)

-Can someone explain to me what am i doing wrong?

**Comment by Arthri** [comment]
Hello! Thank's for creating an issue, but you've deleted the template entirely. It's hard to tell what happened based of a vague error alone. What version of TShock are you using?

**Reply by NerdyTrout** [reply]
I'm using 1.4.3.5 version

**Reply by Arthri** [reply]
Is your client 1.4.3.6 and have you entered it in singleplayer after getting 1.4.3.6?
